README.md

```diff
@@ -7,17 +7,53 @@ Advanced topic group AI: Ket-Hwa, Tingxu, Xiaoting, Zhuoyan
 
 Advanced topic group multiplayer: Jagger, Alli, Sean
 
-# game premise:
+## game premise:
 
 It is a 2D pixel-art platformer where cooperation is necessary. Players control several characters bound by a rope, to conquer the majestic and treacherous slopes of a mysterious mountain, you must master the art of collaborative climbing to reach the top.
 
-# Advanced topic AI: 
+## Advanced topic AI: 
 main implementation: AI created should be a deep learning model focused as a cooperative player for platforming, using a behavior tree to solve jumping with 2 or more players. 
 ### check with team which learning we are doing
 
-# Advanced topic Multiplayer:
+## Advanced topic Multiplayer:
 main implementation: multiplayer should be a client server model
 
-# midterm goal:
+## midterm goal:
 
+AI - AI can do inputs
 
+Multiplayer - connect to a host clien (client and server can communicate)
```

Comment:
typo ("clien")

Need more details. E.g., a single client can connect to the server and send a "Hello" message and receive a response.